doc-Community_Collaboration_Guide/topics/structure.adoc

```diff
@@ -64,7 +64,38 @@ The `ifdef` statement must be at the start of a new line, even for inline refere
 
 === Rendering Content
 
-Placeholder text.
+When your source content is ready to be reviewed or published, you can render it into a display format that can be hosted on a website, included directly in a product, or otherwise accessed by your users. The recommended tool for rendering AsciiDoc into a display format is AsciiDoctor. AsciiDoctor is a fully open-source, Ruby-based implementation of AsciiDoc, and is also the tool that GitHub itself uses to render the content of files written in AsciiDoc.
+
+See the project home page for instructions on how to install AsciiDoctor on a variety of platforms:
+
+* http://asciidoctor.org/
+
+The following is the basic syntax for rendering content into a display format:
+
+[options="nowrap" subs="verbatim,quotes"]
+----
+$ asciidoctor _<main_file>_.adoc
```

Comment:
This should be apparent to users, but by <main_file>  what do we mean? Internally, we call it "master" fairly regularly. In DITA, this file would be referred to as the "master map file" or some such.

Reply:
@sjay72 - Thank you for the review!

That is an excellent point, and I completely agree that this will be unclear to users. I have now changed this to be explicitly 'master.adoc' given that we are talking in the context of our framework, and we will have a brief explanation of this file earlier in the Structure chapter.

What do you think?